README.md

```diff
@@ -259,6 +259,29 @@ GitHub actions will then pick up the branch name and use it to clone that branch
 
 When it's time to merge the sibling PRs, you'll need to merge the prisma/prisma PR first, so when merging the engines PR you have the code of the adapters ready in prisma/prisma `main` branch.
 
+### Testing engines in `prisma/prisma`
+
+You can trigger releases from this repository to npm that can be used for testing the engines in `prisma/prisma` both automatically, or manually:
+
+#### Automated integration releases from this repository to npm
+
+(Since July 2022). Any branch name starting with `integration/` will, first, run the full test suite in Buildkite `[Test] Prisma Engines` and, second, if passing, run the publish pipeline (build and upload engines to S3)
```

Comment:
```suggestion
(Since July 2022). Any branch name starting with `integration/` will, first, run the full test suite in Buildkite `[Test] Prisma Engines` and, second, if passing, run the publish pipeline (build and upload engines to S3 & R2)
```